### ISSUE TITLE
feat(@angular/google-maps): Add Custom HTML Marker Support

### ISSUE THREAD

**Issue by hassang11** [angular/components#21225]
currently @angular/google-maps does not support creating your own html marker .

Use Case
It will be helpful in customizing the marker and we will have more control over it

**Comment by vincentpare1** [comment]
+1

**Comment by laraw** [comment]
Agreed with this. I got a pretty fair way with this but realised it couldn't do what I wanted, which was to be able to be able to map the markers individually to a particular object in an array of objects so that I could display just that objects content. I don't see any way to do this since you can't customise the markers one by one, you can only customise all the markers.

**Comment by danilomr12** [comment]
+2

**Comment by mamzellejuu** [comment]
+1

**Comment by MikeBecatti** [comment]
To solve this problem I used the @googlemaps/markerwithlabel library and passed a reference to map instance retrieved from the mapInitializedEvent.  Good luck!

**Comment by crisbeto** [comment]
I think this should be supported now using the `map-advanced-marker` and the `content` input.
